glean-core/python/glean/util.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import locale
+
+
+def get_locale_tag() -> str:
```

Comment:
Just to double check: the `baseline` ping field this function is populating represents `application locale`, not the system locale. Does this function returns the same? Is there any difference in Python  between app locale and system locale?

Reply:
Python follows POSIX on this, where this is a system locale, and the user can override their locale, and the locale can be overridden anywhere in the process tree.  The Python locale API only gives access to the context in which it is running, so I guess that's basically analogous to "application locale"...